StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthDataSource.swift

```diff
@@ -37,16 +40,22 @@ final class PartnerAuthDataSourceImplementation: PartnerAuthDataSource {
     init(
         institution: FinancialConnectionsInstitution,
         manifest: FinancialConnectionsSessionManifest,
+        returnURL: String?,
         apiClient: FinancialConnectionsAPIClient,
         clientSecret: String,
         analyticsClient: FinancialConnectionsAnalyticsClient
     ) {
         self.institution = institution
         self.manifest = manifest
+        self.returnURL = returnURL
         self.apiClient = apiClient
         self.clientSecret = clientSecret
         self.analyticsClient = analyticsClient
     }
+
+    func getPendingAuthSession() -> FinancialConnectionsAuthSession? {
```

Comment:
made this ugly function cause can't make it get/private(set) in impl when the var is declared get only in protocol

Reply:
I doesn't let me do that :(